features/application/MainNavigation.feature

```diff
@@ -96,7 +96,7 @@ Feature: index
       | $main-nav .nav-item-service-list-view a      | Services           |
       | $main-nav .nav-item-gateway-list-view a      | Gateways           |
       | $main-nav .nav-item-data-plane-list-view a   | Data plane proxies |
-      # TODO: This should say Circuit Breakers
+      # TODO: This should say CircuitBreaker. It currently can’t because the mocks will randomly change which policies exist.
```

Comment:
Q: Just for future knowledge, can't we mock things out so we know the title of the first policy page? (not asking for this to be done here)

Reply:
Huh, this test failed now even though I added the right mock change and ran the test locally 😕

Reply:
Oh I think this is flaky. I got the Zones and CircuitBreaker one failing locally now and on second run they all passed.

Reply:
Okay, that‘s way too flaky, I’m undoing this again. Strange.

Reply:
Just for future anyone reading this, see above comment on this